x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/exceptions/use_import_exception_list.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Just to understand, I just find out that we have `kbn-securitysolution-list-hooks` :D not sure still when to use it!
Do you think we need to place this hook there?

Reply:
That sounds good to me! I can update this PR with those changes now. There is even a TODO to replace this with the kbn version haha.

Reply:
Just looking and that package is specific for the `lists` plugin. So that is for value lists, which are part of exception lists but not for the work we are doing for managing exception lists. Can't move this there unfortunately.

Reply:
thanks for looking into it!